gen_cert.py

```diff
@@ -1901,38 +1919,37 @@ def fontlist_with_style(a_style):
 
         # SECTION: disclaimer text
         print_disclaimer = not self.cert_data.get('HAS_DISCLAIMER', False)
-        disclaimer_text = getattr(settings, 'CERTS_SITE_DISCLAIMER_TEXT', '')
-        if print_disclaimer and disclaimer_text:
+
+        if print_disclaimer and translated_disclaimer:
             (fonttag, fontfile, disclaimer_style) = font_for_string(
                 fontlist_with_style(style_small_text),
-                disclaimer_text,
+                translated_disclaimer,
             )
 
             max_height = disclaimer_style.leading * 3  # allow for up to 9 lines of text
             max_width = MAX_FULL_WIDTH
             yOffset = 89  # distance from bottom of page (in points)
 
-            paragraph = Paragraph(disclaimer_text, disclaimer_style)
+            paragraph = Paragraph(translated_disclaimer, disclaimer_style)
             width, height = paragraph.wrapOn(PAGE, max_width, max_height)
 
             paragraph.drawOn(PAGE, GUTTER_WIDTH, yOffset)
 
         # SECTION: Honor code
         if verify_me_p:
-            paragraph_string = u"Authenticity of this {cert_label} can be verified at " \
-                u"<a href='{verify_url}/{verify_path}/{verify_uuid}'>" \
-                u"<b>{verify_url}/{verify_path}/{verify_uuid}</b></a>"
-
-            paragraph_string = paragraph_string.format(
-                cert_label=self.cert_label_singular,
+            verify_link = u"<a href='{verify_url}/{verify_path}/{verify_uuid}'>" \
```

Comment:
You can drop the `\` line continuation by wrapping the string in parens and relying on Python's built-in string concatenation.

```python
verify_link = (
    u"<a href='{verify_url}/{verify_path}/{verify_uuid}'>"
    u"<b>{verify_url}/{verify_path}/{verify_uuid}</b>"
    "u"</a>"
)